Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -53,6 +53,11 @@ public static void Shutdown()
                     s_ServerHostId = -1;
                 }
 
+                NetworkManager.singleton.transport.OnServerDisconnected.RemoveListener(OnDisconnected);
```

Comment:
Taking another look about this  perhaps this should be in a RegisterHandlers()  method

Reply:
there already is a RegisterMessageHandlers. might be confusing then?

Reply:
true,  that is probably why I did not do it originally.